src/components/Dropdown/Dropdown.tsx

```diff
@@ -253,6 +265,10 @@ export default class Dropdown extends AutoControlledComponent<
           selectedItem={search && !multiple ? undefined : null}
           getA11yStatusMessage={getA11yStatusMessage}
           onStateChange={changes => {
+            if (changes.isOpen !== undefined) {
```

Comment:
let's just make a new function here, this one got bigger.

Reply:
done